rasa/plugin.py

```diff
@@ -70,6 +70,6 @@ def init_managers(endpoints_file: Optional[Text]) -> None:
 def create_tracker_store(  # type: ignore[empty-body]
     endpoint_config: Union["TrackerStore", "EndpointConfig"],
     domain: "Domain",
-    event_broker: Optional["EventBroker"] = None,
+    event_broker: Optional["EventBroker"],
```

Comment:
We should add a test which will validate that event_broker parameter is passed properly.

Reply:
Addressed: https://github.com/RasaHQ/rasa/pull/12230/commits/17dbd8dc95bff815a5679feffbdca26906fb6cdf